data/strings/sound.txt

```diff
@@ -1178,6 +1209,44 @@
     zh-Hans = 然后
     zh-Hant = 然後
 
+  [dist_direction_onto_street]
+    comment = %1$s is the distance like in_500_meters, %2$s is the direction like make_a_right_turn_street or make_a_right_turn if unavailable, and %3$s is the name/number of the street/exit.
+    en = %1$s %2$s onto %3$s
```

Comment:
It looks like %2 can be directly embedded here instead of referencing an external string, right?

Reply:
No, it's replaced by any direction like turn left, turn right, make a slight right turn, take the 3rd exit, etc. We'd have to duplicate this positional format across every permutation and for 90% of languages it's just unnecessary